forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -120,9 +174,12 @@ internal class BTPinCollector(
                 "payment_ref" to paymentRef
             )
         )
+
+        measurement.start()
         val response = runCatching {
             bt.proxy.post(proxyRequest)
```

Comment:
Alternatively, instead of relying on HTTP status codes for tracking error rates, we could use our [error codes](https://docs.joinforage.app/reference/errors#sdk-errors) to distinguish between errors from Forage vs. from Basis Theory.
 
IN Datadog: filter out error codes like ebt_error_51 (insufficient funds), ebt_error_55 from the Vault Proxy error rate monitors + graphs.